services/galley/src/Galley/Intra/Client.hs

```diff
@@ -84,17 +85,28 @@ addLegalHoldClientToUser uid connId prekeys lastPrekey' = do
                   Nothing
                   Nothing
 
+removeLegalHoldClientFromUser :: UserId -> Galley ()
+removeLegalHoldClientFromUser targetUid = do
+    (brigHost, brigPort) <- brigReq
+    void . call "brig"
+        $ method DELETE
+        . host brigHost
+        . port brigPort
+                . paths ["i", "clients", "legalhold", toByteString' targetUid]
+        . contentJson
+        . expect2xx
+
 brigAddClient :: UserId -> ConnId -> NewClient -> Galley Client
 brigAddClient uid connId client = do
     (brigHost, brigPort) <- brigReq
     r <- call "brig"
         $ method POST
         . host brigHost
         . port brigPort
-        . header "Z-User" (toByteString' uid)
```

Comment:
oops  :)

Reply:
This is intentionaly; we're calling an internal endpoint now, so we don't need the Z-User header. I can leave it if you think we should still have it?

Reply:
ah, i misread.  i agree it shouldn't be sent, just thought that this handler was always an internal endpoint and this was there by mistake until now.

anyway, please disregard this thread.  all good!